packages/components/molecules/f-media-element/src/components/MediaElement.vue

```diff
@@ -1,18 +1,120 @@
 <template>
     <div
-        :class="$style['c-mediaElement']"
-        data-test-id="mediaElement-component">
-        {{ componentName }}
+        data-test-id="mediaElement-component"
+        :class="[$style['c-mediaElement'], { [$style['c-mediaElement--stack']]: stacked }, { [$style['c-mediaElement--reverse']]: reverse }]">
+        <div :class="[$style['c-mediaElement-content'], contentAlignClass]">
+            <h3 :class="[$style['c-mediaElement-title'], $style['c-mediaElement-text'], textSizeClass]">
```

Comment:
I find these easier to parse if they're on separate lines:

```suggestion
            <h3 :class="[
                $style['c-mediaElement-title'],
                $style['c-mediaElement-text'],
                textSizeClass]">
```

Reply:
agreed will make the suggested change

Reply:
fixed